### ISSUE TITLE
No named parameter with the name textInputAction

### ISSUE THREAD

**Issue by ratsey** [codegrue/card_settings#16]
Getting a compilation failure with:

```
compiler message: file:///E:/flutter/.pub-cache/hosted/pub.dartlang.org/card_settings-0.1.11/lib/widgets/text_fields/card_settings_text.dart:69:17: Error: No named parameter with the name 'textInputAction'.
compiler message:                 textInputAction: textInputAction,
compiler message:                 ^^^^^^^^^^^^^^^
compiler message: file:///E:/flutter/packages/flutter/lib/src/material/text_field.dart:99:9: Context: Found this candidate, but the arguments don't match.
compiler message:   const TextField({
```
card_settings_-0.1.11

[√] Flutter (Channel beta, v0.5.1, on Microsoft Windows [Version 10.0.17134.228], locale en-US)
[√] Android toolchain - develop for Android devices (Android SDK 27.0.3)
[√] Android Studio (version 3.1)
[√] IntelliJ IDEA Community Edition (version 2018.1)
[!] VS Code, 64-bit edition (version 1.24.1)
[√] Connected devices (1 available)

**Comment by codegrue** [comment]
The project is built on the master branch so it looks like they changed that property in beta. Maybe it can be removed completely.

**Comment by codegrue** [comment]
Fixed in 0.1.12
